packages/ui/components/Pages/Program/MainSection.tsx

```diff
@@ -23,28 +33,28 @@ const MainSection = ({
   return (
     <main className="bg-primary">
       <div className="py-6 px-6 flex flex-col mb-4">
-        <div className="flex border border-gray-400 w-36">
-          <Button
+        <div>
+          {/* <Button
```

Comment:
Can you please remove this and other commented out sections unless you plan on using them in the future.